### DISCUSSION TITLE
Add the option to display charts by Year, Month, Weekly, Day

### DISCUSSION THREAD

**Discussion by alexvolte** [plausible/analytics#2456] [Anything Else]
Would it be possible to add the option to display charts by Year, Month, Weekly, Day on the choosing date panel?

**Comment by metmarkosaric** [comment]
hi @alexvolte! yes, we're already testing this feature and it will be the next big feature we release. stay tuned! #1574

**Comment by metmarkosaric** [comment]
**UPDATE:** You can now set a different interval on the top graph to get a deeper level of detail and make it easier to spot trends.
